pkg/tasks/prometheus.go

```diff
@@ -233,8 +233,37 @@ func (t *PrometheusTask) Run() error {
 		}
 	}
 
+	// Create trusted CA bundle ConfigMap.
+	trustedCA, err := t.factory.AlertmanagerTrustedCABundle()
```

Comment:
Hmm why the AlertmanagerTrustedCABundle here?

Reply:
whops, copy pasta